chain/chain/src/chain.rs

```diff
@@ -106,6 +106,16 @@ pub const NUM_EPOCHS_TO_KEEP_STORE_DATA: u64 = 5;
 /// Maximum number of height to go through at each step when cleaning forks during garbage collection.
 const GC_FORK_CLEAN_STEP: u64 = 1000;
 
+/// Grab the acceptable time difference. This may differ between sandbox and regular networks
+const fn acceptable_time_difference() -> i64 {
```

Comment:
move it to line 94 - and do a simpler thing:

#[cfg(feature = "sandbox")]
ACCEPTABLE_TIME_DIFFERENCE = 60 * 60 * 24 * 365 * 10000;
#[!cfg(not(feature = "sandbox"))]
ACCEPTABLE_TIME_DIFFERENCE = 12*10;

Reply:
a bit opinionated but had the function instead since we can have it all in one place. But it's not a big deal